lib/rubocop/cop/style/extra_spacing.rb

```diff
@@ -67,11 +67,14 @@ def check_tokens(ast, t1, t2)
         end
 
         def check_assignment(token)
-          # minus 2 is because pos.line is zero-based
-          line = processed_source.lines[token.pos.line - 2]
+          preceding = @asgn_lines.include?(token.pos.line - 1)
+          line = if preceding
```

Comment:
what line? :-)

I'm guessing that in the interest of clarity some helper method can be extracted here.